readme.md

```diff
@@ -7,7 +7,8 @@ Flow Navigator provides a simplified API for managing navigation flows in your R
 
 ## Features
 
-- **Simplified Flow Management**: Easily manage the navigation flow without the need for screens to be aware of their position in the flow.
+- **Simplified Flow Management**: Get a comprehensive overview of your flow in a single location. This allows new developers to quickly understand the entire flow without the need to examine each page individually.
+- **Separation of responsibility**: Screens are not aware of their specific position in the flow.They don't need to know which page is next; they simply navigate to the next page in the flow.
```

Comment:
il manque un espace avant le "They don't need to know which" 😅

Reply:
t'es fort